pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
If this is a completely separate thing then it should probably just rely on `mantid` being installed and available on the python path. Or do you mean from a developer perspective?